node-test/src/test/scala/org/bitcoins/node/NeutrinoNodeTest.scala

```diff
@@ -34,15 +31,36 @@ class NeutrinoNodeTest extends NodeTestWithCachedBitcoindNewest {
 
   behavior of "NeutrinoNode"
 
+  it must "be able to connect and disconnect from all peers" in {
+    nodeConnectedWithBitcoind: NeutrinoNodeConnectedWithBitcoinds =>
+      //checking all peers are connected
+      val node = nodeConnectedWithBitcoind.node
+      val connFs = node.peers.indices.map(node.isConnected)
+      val connF = Future.sequence(connFs).map(_.forall(_ == true))
+      connF.map(assert(_))
```

Comment:
This is a small bug, this is commonly called "Dropping a future on the floor". 

Since this computation is executed asynchronously, you don't have a good idea of when this will actually run. It could be after the entire test case is done! 

You need to incorporate this result into the final future you return from this test case (`disconnF`) so that the results are incorporated into that `Future`. 

You can fix this by 

```scala
...
val assertion1F = connF.map(assert(_))
...
val disconnF = for {
  _ <- assertion1F 
  _ <- node.stop()
  f  <- isAllDisconnectedF 
} yield f
disConnF.map(assert(_))
```

Reply:
Sorry, my bad, missed that while changing 😓.